savant/api/builder.py

```diff
@@ -67,18 +79,16 @@ def add_objects_to_video_frame(
         frame.add_object(video_object, IdCollisionResolutionPolicy.Error)
         track_id = video_object.get_track_id()
```

Comment:
@tomskikh , it doesn't look correct. The parent is not related to the track.

The object in metadata must have a reference to the parent. it is not about the track.